### PR TITLE
Updated rules content

### LINKED ISSUE #80: Update Rules (Content)
New rules without images. New illustrations will follow in a future update.

### Kommunikation
Gute Kommunikation ist essentiell für eine funktionierende Critical Mass. Viele Teilnehmer sind zum ersten mal dabei. Sprecht miteinander und erklärt ihnen die Verhaltensregeln. Informiert eure Mitfahrer wenn abgebogen oder gehalten wird. Warnt euch gegenseitig bei Gefahren und bedankt euch wenn Leute Kreuzungen sichern (korken).
### Communication
Good communication is essential for a proper Critical Mass. Remember that many of your fellow riders are there for the first time. Reach out and explain the rules of the mass if anyone seems unclear. Let people know when you are stopping or turning. Warn riders of any danger and give some thanks to those who are protecting the intersections (corking).

### Hab Spass
Die Critical Mass sollte sicher sein und Spass machen. Manche Fahrer werden wütend und aggressiv. Erklärt das Konzept umstehenden und macht Verbündete aus ihnen. Behaltet ein ruhiges Gemüt und lasst aggressive Situationen nicht eskalieren.
### Good Vibes
The Critical Mass should be fun and safe for everyone. Some drivers get angry and aggressive. Try your best to explain the Critical Mass to bystanders and make allies out of them. Never escalate any aggression and always keep the situation chill.

### Gegenverkehr
Die Critical Mass ist als ein Fahrzeug anzusehen. Bleibt auf eurer Fahrspur und fahrt nicht in den Gegenverkehr! Damit bringt ihr euch und andere Verkehrsteilnehmer in Gefahr.
### Oncoming Traffic
The Critical Mass should look and act like one unified vehicle driving on the street. Stick to your own lane and never ride into the opposite lane of oncoming traffic. This can be incredibly dangerous for riders in the mass. 

### Geschwindigkeit
Damit die Critical Mass nicht auseinander gerissen wird, ist es wichtig, ein gemäßigtes Tempo beizubehalten und vorne nicht zu schnell zu fahren. Das verhindert das Bilden von Lücken. Wenn doch mal eine Lücke entsteht, so versucht aufzuschließen!
### Pace Yourself
Keeping an even pace is essential to keeping one unified mass and avoiding any interference from competing traffic. To avoid cracks in the mass, don't ride too fast. If you see too much space starting to form, try to fill in immediately. 

### Geschlossener Verband
Fahrt an Kreuzungen nie zwischen stehenden Autos hindurch, sondern wartet dahinter. Sollte doch mal ein Fahrzeug in der Critical Mass stecken, leitet es hinaus. Das Blockieren verursacht nur mehr Stau und stört die Weiterfahrt. Als Kopf der Critical Mass darfst du nur bei Grün über die Ampel. Sobald der Verband einmal die Kreuzung überquert hat, müssen alle folgenden Radfahrer auch bei Rot weiterfahren.
### Unified Mass
Never ride between standing cars at intersections. If a car gets itself stuck in the mass, just let it out. Blocking cars only causes more traffic to back up. Those leading the Critical Mass should only pass a traffic light if it is green. Once the mass is moving through an intersection, others should continue to follow, even when the light has turned red.  

### Blaulicht
Blockiert niemals Krankenwagen oder andere Einsatzfahrzeuge! Macht Platz, fahrt an den Rand und bildet eine Gasse. Prüft eure Umgebung und fahrt erst weiter wenn das Weiterkommen der Einsatzfahrzeuge gewährleistet ist.
### Emergency Vehicle
Never block an emergency vehicle! Ride along the edges and create a lane where the vehicle can pass. Be absolutely sure the vehicle has safely passed before riding as usual.  

### Kreuzungen (Korken)
Korken ist das Absichern der Critical Mass an Kreuzungen. Stellt euch mit eurem Rad direkt vor die Autos und versperrt ihnen den Weg in die Critical Mass. Ein Fahrrad ist meistens nicht genug. Arbeitet zusammen um die Kreuzungen gemeinsam mit anderen Radfahrern zu sichern und löst euch gegenseitig ab. Manche Autofahrer reagieren uneinsichtig und aggressiv. Haltet an und helft die Situation zu schlichten. Klärt sie ruhig und sachlich über die StVo und Critical Mass auf.
### Intersections (Corking)
Corking is protecting the Critical Mass at intersections. Stand at the intersection and use your bicycle to block cars from driving into the mass. Sometimes one bicycle is not enough. Work together to block the intersections with many bicycles and relieve those who have been corking for a long time. Some drivers get angry and aggressive for being blocked. Stop and help to calm the situation down. Explain the Critical Mass and traffic regulations calmly and factually.

### Bürgersteig
Fahrt nicht auf dem Bürgersteig! Als schwächstes Glied im Netz der Verkehrsteilnehmer gilt den Fußgängern gegenüber besondere Rücksicht. Reclaim the Streets, nicht die Gehwege.
### Sidewalk
Never ride on the sidewalk! Pedestrians are the weakest link in the traffic chain, and we need to look out for them. Reclaim the streets, not the sidewalk.  

### Polizeibegleitung
Oft wird die Critical Mass von der Polizei begleitet. Sie sichern kritische Kreuzungen und zähmen wütende Autofahrer. Als Kopf der Critical Mass ist es vorteilhaft (je nach Laune und Situation) zu kooperieren und  die gewünschte Route der Polizei mitzuteilen.
### Police Escort
The Critical Mass will be escorted by the police quite often. They protect intersections and tame raging drivers. As the head of the Critical Mass it can be an advantage (depending on your mood and situation) to cooperate and communicate the desired route to the police.

### PR DESCRIPTION
**Pull request by lennet**
missing:
- [ ] Illustrations
- [ ] Italian translation
- [ ] French translation

fixes #80

### CODE REVIEW

**Comment by mltbnz** [comment]
Looks good.
Though neither english or italian is a native language to me.

File: CriticalMass/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -30,49 +30,61 @@
...
-"rules.text.green" = "When you arrive at a red light while driving as part of the head of the mass, be sure to stop when the traffic light shows red.";
+"rules.title.pace" = "Pace Yourself";
 
 /* No comment provided by engineer. */
-"rules.text.haveFun" = "Enjoy reclaimed streets. Check out the Sound Bikes. Chat with motorists and pedestrian to let them know what's going on.\n\nAnd most importantly: Have fun!";
+"rules.title.mass" = "Unified Mass";
 
 /* No comment provided by engineer. */
-"rules.text.stayLoose" = "Don't allow yourself to be provoked. Be friendly to the police, motorists and everybody else, even if they are not.";
+"rules.title.emergency" = "Police Escort";
```
Comment (zutrinken) [inline CriticalMass/en.lproj/Localizable.strings]: 🚑 Correct title is "Emergency Vehicle"

File: CriticalMass/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -30,49 +30,61 @@
...
-"rules.text.green" = "Se sei in testa al gruppo ed incontri un semaforo rosso, devi attendere che diventi verde.";
+"rules.title.pace" = "Pace Yourself";
 
 /* No comment provided by engineer. */
-"rules.text.haveFun" = "Goditi le strade senza auto. Dai un'occhiata alle Sound Bikes. Scambia quattro chiacchere con automobilisti e pedoni, spiegando cosa sta succendendo.\n\nE la cosa più importante: divertiti!";
+"rules.title.mass" = "Unified Mass";
 
 /* No comment provided by engineer. */
-"rules.text.stayLoose" = "Non farti provocare. Sii amichevole con la polizia, gli automobilisti e chiunque altro, anche nel caso loro non lo fossero.";
+"rules.title.emergency" = "Police Escort";
```
Comment (zutrinken) [inline CriticalMass/it.lproj/Localizable.strings]: also "Emergency Vehicle" (for the moment)

**Comment by mltbnz** [comment]
Is this one still blocked/waiting by/for something?

**Comment by lennet** [comment]
Italian and French translation are missing.. I can ask a friend for the French translations

**Comment by mltbnz** [comment]
Ok. I will ask a friend for italien 🇮🇹

**Comment by mltbnz** [comment]
Btw.: Couldn't we also use the localization system that Android has in place? Would make translation easier for non-gitHub people and we would get some additional languages for free 🐳

**Comment by lennet** [comment]
Yes we should do that. Created an issue for that #126

File: CriticalMass/it.lproj/Localizable.strings (hunk 2 above)
Comment (StartingCoding) [reply CriticalMass/it.lproj/Localizable.strings]: Emergency Vehicle could be "Veicoli d'Emergenza"

**Comment by lennet** [comment]
Yeah let’s close it and restart the Initiative of updating the rules

**Pull request closed without merging**